### DISCUSSION TITLE
评论区“UP主觉得很赞”没有夜间模式

### DISCUSSION THREAD

**Discussion by CuewarsTaner** [the1812/Bilibili-Evolved#4844] [功能建议 / Ideas]
![image](https://github.com/user-attachments/assets/2c1d353a-41e0-4f4c-89f8-3194412c442f)
如图，文字底色是白色的，在夜间模式里有些突兀。希望可以做成暗灰色底框+黑色文字。

**Comment by Geeyun-JY3** [comment]
看置顶：#4843
